adm_program/system/function.php

```diff
@@ -841,3 +794,53 @@ function replaceValuesArrWithQM(array $valuesArray)
 {
     return implode(',', array_fill(0, count($valuesArray), '?'));
 }
+
+/**
+ * Berechnung der Maximalerlaubten Dateiuploadgröße in Byte
+ * @deprecated 3.3.0:4.0.0 "admFuncMaxUploadSize()" is a typo. Use "PhpIni::getUploadMaxSize()" instead.
+ * @return int
+ */
+function admFuncMaxUploadSize()
+{
+    global $gLogger;
+
+    $gLogger->warning('DEPRECATED: "admFuncMaxUploadSize()" is deprecated, use "PhpIni::getUploadMaxSize()" instead!');
+
+    return PhpIni::getUploadMaxSize();
+}
+
+/**
+ * @deprecated 3.3.0:4.0.0 "admFuncGetBytesFromSize()" is deprecated, there is no replacement.
+ * @param string $data
+ * @param bool   $decimalMulti
+ * @return int
+ */
+function admFuncGetBytesFromSize($data, $decimalMulti = false)
```

Comment:
Why not call the new method of the PhpIni class within this function? So we don't have the code twice.

Reply:
because the function in PhpIni is private and so this doesn't work. and i want to leave the function private